### ISSUE TITLE
Admin UI isn't enabled by default

### ISSUE THREAD

**Issue by allen13** [influxdata/influxdb-docker#11]
Either add this to the base config:

```
[admin]
  enabled = true
  bind-address = ":8083"
  https-enabled = false
  https-certificate = "/etc/ssl/influxdb.pem"
```

Or add enviornment variable to dockerhub documentation:

-e INFLUXDB_ADMIN_ENABLED=true

Or remove -p 8083:8083 from all of the examples as it is misleading

**Comment by jsternberg** [comment]
It looks like there are some other default settings that don't get set correctly if a config file is provided. I'll work on this and try to update the image.

**Comment by jsternberg** [comment]
Fixed in e5f7420. I've submitted #1741 to update the hosted image to the latest.
